packages/syntax-core/src/TextArea/TextArea.tsx

```diff
@@ -88,6 +95,11 @@ const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>(
     const disabled = !isHydrated || disabledProp;
     const reactId = useId();
     const inputId = id ?? reactId;
+    const textColor = on === "darkBackground" ? "white" : "gray900";
```

Comment:
the original inlined color  was `gray700`, is this intentional?

Reply:
Yep! These should all be gray900 and match what we have in `TextField`